frontend/src/mlmd/MlmdUtils.ts

```diff
@@ -287,6 +287,13 @@ export interface LinkedArtifact {
   artifact: Artifact;
 }
 
+export function getStoreSessionInfoFromArtifact(artifact: LinkedArtifact): string | undefined {
+  return artifact.artifact
+    .getCustomPropertiesMap()
+    .get('store_session_info')
```

Comment:
If I understand correctly, the store_session_info here is a string that specifies the `providerInfo`, which contains secret name and token key.. I wonder if this is a security concern to expose it on the UI. It might be better to only specify the configmap/secret name which contains sensitive information in the cluster, so only the control plane can access it via such a hint.

Reply:
@zijianjoy this sessioninfo can be thought of as a pointer to a secret in the cluster, it consist of: 

* secretname
* secretkey  (not the gcs token or s3 access credentials them selves, but the k8s notion of a key) 

an example of what might be stored in mlmd as an artifact property is: 

for gcs:
```json
{
  "Provider": "gs",
  "Params": {
    "fromEnv": "false",
    "secretName": "gcs-secret",
    "tokenKey": "credentials.json"
  }
}
```

for aws s3: 

```json
{
  "Provider": "s3",
  "Params": {
    "accessKeyKey": "customaccessKey",
    "disableSSL": "false",
    "endpoint": "s3.amazonaws.com",
    "fromEnv": "false",
    "region": "us-east-2",
    "secretKeyKey": "customsecretKey",
    "secretName": "s3secret"
  }
}
```


more details on this backend implementation are described here if you are [curious](https://github.com/kubeflow/pipelines/pull/10625).